o/olombelo-ricky.md

```diff
@@ -0,0 +1,9 @@
+# Olombelo Ricky
+
+![Olombelo Ricky](olombelo.jpg)
+
+Olombelona Ricky, also known as Ricky, is a Malagasy vocalist and roots musician based in Antananarivo, Madagascar. Launching his musical career in 1991, Ricky recorded his first album in 1998, sparking the phenomenon of "Rickymania". His socio-political messages and uncompromising approach to his career have earned him a strong following among urban youth within Madagascar. He is esteemed as a musical and cultural ethnologist by many of his fellow Malagasy musicians for his effort to capture and reinvent the traditional musical heritage of the island and that of the near-mythical Vazimba people in particular, believed by many Malagasy to be the island's earliest inhabitants. Despite an uncompromising approach to his career which slowed his international exposure, Ricky has performed at numerous international music festivals over the course of his career and regularly tours at home and abroad.
+
+**Genre:** Jazz
+
+**Artist homepage:** [OLombelo Ricky](https://www.last.fm/fr/music/Olombelo+Ricky/+wiki)
```

Comment:
Fix the capitalisation of the `L`

Reply:
I think I've made the change, please check.